cli/app.go

```diff
@@ -21,6 +21,9 @@ If a name is not provided, a random name will be generated.
 If run from a git repository, a 'flynn' remote will be created or replaced that
 allows deploying the application via git.
 
+Options:
+	-r, --remote <remote>  Name of git remote on local repo.
```

Comment:
Do you mean skip [this change](https://github.com/flynn/flynn/pull/717/files#diff-333b1ce61f983aec7830fc16c0445bcaR98)?